app/components/confirmation/funding/view.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+module Confirmation
+  module Funding
+    class View < GovukComponent::Base
+      include SanitizeHelper
+      include FundingHelper
+
+      attr_accessor :data_model
+
+      def initialize(data_model:)
+        @data_model = data_model
+      end
+
+      def trainee
+        data_model.is_a?(Trainee) ? data_model : data_model.trainee
+      end
+
+      def title
+        t(".title")
+      end
+
+      def rows
+        rows = [
+          {
+            key: t(".training_initiative"),
+            value: training_initiative,
+            action: action_link("training initiative", edit_trainee_funding_training_initiative_path(trainee)),
+          },
+        ]
+
+        if show_bursary_funding?
+          rows << {
+            key: t(".bursary_funding"),
+            value: bursary_funding,
+            action: action_link("bursary funding", edit_trainee_funding_bursary_path(trainee)),
+          }
+        end
+
+        rows
+      end
+
+    private
+
+      def course_subject_one
+        trainee.course_subject_one
+      end
+
+      def show_bursary_funding?
```

Comment:
**This logic needs updating to match the prototype:**

1. If there is no bursary available for the route i.e. `!CalculateBursary.available_for_route?(route)`, then this row is missing.

2. If there is a bursary available for this route i.e. `CalculateBursary.available_for_route?(route)`, then this row is present.

If (2) and the user has selected `applying_for_bursary`, then show the amount as you've done here.

If (2) and the user has not selected `applying_for_bursary`, then show "Not bursary funded" i.e. there was a bursary available but the user has decided that they don't qualify.

<img width="1053" alt="Screenshot 2021-06-25 at 14 29 31" src="https://user-images.githubusercontent.com/18436946/123431992-caa2bf80-d5c1-11eb-8f5d-5d4c32795e28.png">

Reply:
At the moment, you're showing the bursary row and a bursary amount even if the user has selected not applying for a bursary.